mnefun/_mnefun.py

```diff
@@ -250,6 +250,16 @@ class Params(Frozen):
         Default is 7 mm. Defines source grid spacing for volumetric source
         space.
 
+    Attributes
+    ----------
+    on_missing : string
+        Can set to ‘error’ | ‘warning’ | ‘ignore’. Default is 'error'. Determine
+        what to do if one or several event ids are not found in the recording
+        during epoching. See mne.Epochs docstring for further details.
+    compute_rank : bool
+        Default is False. Set to True to compute rank of the noise covariance
+        matrix during inverse kernel computation.
+
```

Comment:
Is it necessary to redefine parameters as `self.param = arg` after it's been defined in `__init__`? e.g., `self.ssp_eog_reject = ssp_eog_reject`

Reply:
It is necessary to have this assignment in the code, yes, but it only needs to be documented in the docstring in the Parameters section here.

Reply:
Why are the assignments necessary?

Reply:
If the line `self.ssp_eog_reject = ssp_eog_reject` were omitted, there would be no `params.ssp_eog_reject`. The instance only gets attributes that you assign to it.
```
>>> class Dummy(object)
...    def __init__(self, param)
...        pass
...
>>> d = Dummy('foo')
>>> d.param
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Dummy' object has no attribute 'param'
```